ansible/roles/stack-sunbird/templates/sl-assessments.env

```diff
@@ -0,0 +1,97 @@
+# Service Config
+# Port on which the service will listen to
+PORT={{sl_assessments_port | default("7201")}} 
+
+IMPROVEMENT_PROJECT_BASE_URL
```

Comment:
if it's a comment, should contain # at the beginning.

Reply:
improvement project base removed which is defined with out variable